tests/unit-tests/VolleyM.Domain.IdentityAndAccess.UnitTests/IdentityAndAccessStepsBase.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using TechTalk.SpecFlow;
+using VolleyM.Domain.IdentityAndAccess.UnitTests.Fixture;
+using VolleyM.Domain.UnitTests.Framework;
+using VolleyM.Infrastructure.Bootstrap;
+using VolleyM.Infrastructure.IdentityAndAccess.AzureStorage;
+
+namespace VolleyM.Domain.IdentityAndAccess.UnitTests
+{
+    public class IdentityAndAccessStepsBase : SpecFlowBindingBase
+    {
+        protected IIdentityAndAccessFixture Fixture { get; private set; }
+
+        [BeforeTestRun(Order = ONETIME_DOMAIN_FIXTURE_ORDER)]
+        public static void OneTimeSetup()
+        {
+            OneTimeFixtureCreator = CreateOneTimeTestFixture;
+            SpecFlowBindingBase.BeforeTestRun();
+        }
+
+        [AfterTestRun]
+        public static void OneTimeTearDown()
+        {
+            SpecFlowBindingBase.AfterTestRun();
+        }
+
+        private static IOneTimeTestFixture CreateOneTimeTestFixture(TestTarget target)
+        {
+            return target switch
+            {
+                TestTarget.Unit => NoOpOneTimeTestFixture.Instance,
+                TestTarget.AzureCloud => new AzureCloudIdentityAndAccessOneTimeFixture(),
+                TestTarget.OnPremSql => throw new NotSupportedException(),
+                _ => throw new ArgumentOutOfRangeException(nameof(target), target, null)
+            };
+        }
+
+        public override void BeforeEachScenario()
+        {
+            base.BeforeEachScenario();
+
+            Fixture = CreateTestFixture(Target);
+            Fixture.Setup();
+        }
+
+        private IIdentityAndAccessFixture CreateTestFixture(TestTarget target)
+        {
+            return target switch
+            {
+                TestTarget.Unit => (IIdentityAndAccessFixture)new UnitTestIdentityAndAccessFixture(Container),
+                TestTarget.AzureCloud => new AzureCloudIdentityAndAccessFixture(Container),
+                TestTarget.OnPremSql => throw new NotSupportedException(),
+                _ => throw new ArgumentOutOfRangeException(nameof(target), target, null)
+            };
+        }
+
+        protected override IEnumerable<IAssemblyBootstrapper> GetAssemblyBootstrappers()
+        {
+            var result = new List<IAssemblyBootstrapper> { new DomainIdentityAndAccessAssemblyBootstrapper() };
+
+            if (Target == TestTarget.AzureCloud)
+            {
+                result.Add(new InfrastructureIdentityAndAccessAzureStorageBootstrapper());
+            }
+
+            return result;
+        }
```

Comment:
How-To: Step 5: Add bootstrappers which are used by your tested code in production to properly hook up IoC and Automapper